config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -6,6 +6,13 @@ presubmits:
     always_run: false
     run_if_changed: '^(test/e2e/network/|pkg/apis/networking)'
     optional: true
+    decorate: true
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
```

Comment:
why do you need to pull k8s.io/test-infra?

Reply:
Because of `/workspace/scenarios/kubernetes_e2e.py` which is in test-infra repo :)

Reply:
Wait, I'm trying to remember why actually we used this, and maybe wasn't because of the scenarios, but because of https://github.com/kubernetes/test-infra/commit/67e14ce72e5f63600e02b40b9780fbf7e49c7ccd#diff-9495ba672589316a9265fd090ecb9c3c74fd414a11386acafda27df4bd409409

As we do not use anything at test-infra I'll remove it

Reply:
Also, I'm wondering if --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-ubuntu-gce-network-policies is necessary or if this is also covered by podutils, BUT I think for now will leave this way